mozilla/blurts-server
# MNTOR-3261/instrument backend glean

# References:

Jira: MNTOR-3261



# Description

Simple implementation for Glean backend, using similar API to the frontend `record()`.

NOTE: this is branched from https://github.com/mozilla/blurts-server/pull/4544 and is currently blocked from landing on it, but I think we can make progress in the meantime.

# How to test

Create or delete an account, it will be logged. In GCP environments, it will be treated as a structured log message and should be routed to the Mozilla Telemetry infra.

# Checklist (Definition of Done)
